examples/rsp-next-ts/package.json

```diff
@@ -17,15 +17,15 @@
     "@react-spectrum/toast": "^3.0.0-alpha.0",
     "@spectrum-icons/illustrations": "^3.5.0",
     "@spectrum-icons/workflow": "^4.0.3",
-    "next": "^13.1.1",
+    "next": "13.2.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@types/node": "^18.8.2",
     "@types/react": "^18.0.21",
     "eslint": "^8.24.0",
-    "eslint-config-next": "^13.1.1",
+    "eslint-config-next": "13.2.4",
```

Comment:
what is this one changed for?

Reply:
no particular reason other than to keep it in lock step with the nextjs version